src/middlewared/middlewared/plugins/truenas.py

```diff
@@ -114,6 +114,7 @@ async def accept_eula(self):
     async def unaccept_eula(self):
         with open(EULA_PENDING_PATH, "w"):
             pass
+        os.chmod(EULA_PENDING_PATH, 0o600)
```

Comment:
I don't think this should be an async coroutine since it's only doing file IO.

Reply:
Some other methods were async as well in this file - i fixed those too ✅